### DISCUSSION TITLE
How to get the execution time of each SQL operator?

### DISCUSSION THREAD

**Discussion by artorias1024** [databendlabs/databend#9004] [Q&A] [answered]
<img width="1331" alt="image" src="https://user-images.githubusercontent.com/82564604/204462133-73f43e7d-e16b-41a5-b825-18208e5f6114.png">

this is hologres execution graph 
<img width="1380" alt="image" src="https://user-images.githubusercontent.com/82564604/204462520-aaa7efb8-a169-4df6-9d02-0124947ed476.png">

**Comment by sundy-li** [comment, marked as answer]
it's on progress https://github.com/datafuselabs/databend/issues/4238
